PORTAL/jobs/_pyperformance.py

```diff
@@ -2606,7 +2625,10 @@ def add(self, results, *,
         )
         added = list(added)  # Force the iterator to complete.
         index = self._resultsdir.load_index()
-        readme = self._update_table(index)
+        readme = self._update_table(index.summarized(), "README.md")
+        benchmark_results_readme = self._update_table(
+            index, "benchmark-results/README.md"
+        )
```

Comment:
This should make use of `self.datadir`.

```suggestion
        if self.datadir:
            readmes = [
                self._update_table(index.summarized(), "README.md"),
                self._update_table(index, os.path.join(self.datadir, "README.md"))
            )
        else:
            readmes = [
                self._update_table(index, "README.md"),
            ]
```